.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Build medic-couch2pg and run unit test cases
+
+on: [ push, pull_request ]
+
+jobs:
+  build-and-test:
+    name: Build medic-couch2pg
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [ 8, 10, 12 ]
+
+    steps:
+      - name: Login to Docker Hub
```

Comment:
Hi @garethbowen, I'd  like to add the login to avoid reaching the pull limit of Docker Hub, but it's failing. 
I tried doing the travis way like [this](https://github.com/medic/medic-couch2pg/blob/51b296c61d1737951af46220fb4e732d461bdab3/.travis.yml#L12) but it didn't work. I'm wondering if there's a problem in the secrets @ Repository's settings but I'm not admin to check it. 
Can you please look at the settings? 🙏
The rest is working fine, this is the only piece blocking me.